ras_party/views/account_view.py

```diff
@@ -73,3 +73,10 @@ def put_email_verification(token):
 def resend_verification_email(party_uuid):
     response = ras_party.controllers.account_controller.resend_verification_email(party_uuid)
     return make_response(jsonify(response), 200)
+
+
+@account_view.route('/respondents/add_survey', methods=['POST'])
+def respondent_add_survey():
+    payload = request.get_json() or {}
+    response = ras_party.controllers.account_controller.add_new_survey_for_respondent(payload)
```

Comment:
add_new_survey_for_respondent is not returning anything in this pr, not sure if the returning case_id is part of another pr or not.

Reply:
The case id is being queried in the frontstage-api so frontstage is now having this returned from there.